doc/source/ray-metrics.rst

```diff
@@ -0,0 +1,132 @@
+Monitoring Ray
+==============
```

Comment:
Using Ray and Prometheus
========================

Reply:
Maybe `Ray Monitoring with Prometheus`? (I think it is important to add a keyword monitoring)